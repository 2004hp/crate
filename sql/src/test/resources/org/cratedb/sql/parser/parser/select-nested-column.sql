select a['b']['d'] where a['b']['c']['name'] = 'crate'