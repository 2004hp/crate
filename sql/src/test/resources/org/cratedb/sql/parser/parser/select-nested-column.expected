org.cratedb.sql.parser.parser.CursorNode@4c7da1fc
name: null
updateMode: UNSPECIFIED
statementType: SELECT
resultSet: 	
	org.cratedb.sql.parser.parser.SelectNode@4c15ce9f
	isDistinct: false
	resultColumns: 		
		org.cratedb.sql.parser.parser.ResultColumnList@2bb04fbf

		[0]:		
		org.cratedb.sql.parser.parser.ResultColumn@44439b01
		exposedName: a.b.d
		name: a.b.d
		tableName: null
		isDefaultColumn: false
		type: null
		expression: 			
			org.cratedb.sql.parser.parser.NestedColumnReference@327581aa
			columnName: a.b.d
			tableName: null
			type: null
			path:
				[0]:				
				org.cratedb.sql.parser.parser.CharConstantNode@4093d9c7
				value: b
				type: CHAR(1) NOT NULL
				[1]:				
				org.cratedb.sql.parser.parser.CharConstantNode@2abeb8f0
				value: d
				type: CHAR(1) NOT NULL
	fromList: 		
		org.cratedb.sql.parser.parser.FromList@4f5faa7c

	whereClause: 		
		org.cratedb.sql.parser.parser.BinaryRelationalOperatorNode@11b1db9a
		operator: =
		methodName: equals
		type: null
		leftOperand: 			
			org.cratedb.sql.parser.parser.NestedColumnReference@33c86489
			columnName: a.b.c.name
			tableName: null
			type: null
			path:
				[0]:				
				org.cratedb.sql.parser.parser.CharConstantNode@294078ee
				value: b
				type: CHAR(1) NOT NULL
				[1]:				
				org.cratedb.sql.parser.parser.CharConstantNode@47c3666a
				value: c
				type: CHAR(1) NOT NULL
				[2]:				
				org.cratedb.sql.parser.parser.CharConstantNode@6d78ddf4
				value: name
				type: CHAR(4) NOT NULL
		rightOperand: 			
			org.cratedb.sql.parser.parser.CharConstantNode@1a33f652
			value: crate
			type: CHAR(5) NOT NULL